### PR TITLE
[Doc] Update An Image Link for OceanBase Cloud

### PR DESCRIPTION
**Pull request by kyle-oceanbase**
### What does this PR do?

Update Definition Link in oceanbasecloud_overview.json for  OceanBase Cloud.

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
- [ ] If this PR includes a log pipeline, please add a description describing the remappers and processors. 

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: oceanbasecloud/assets/oauth_clients.json [hunk 1]
```diff
@@ -5,10 +5,11 @@
     ],
     "client_role": "integration",
     "name": "oceanbasecloud",
-    "onboarding_url": "https://cloud.oceanbase.com/integrations/Datadog",
+    "onboarding_url": "https://console-en.oceanbase.com/integrations/Datadog",
```
Comment (JoshPatel13) [inline oceanbasecloud/assets/oauth_clients.json:8]: Hi Team, This link doesn't seem to load. Is it working on your end?

Comment (sunzhaoyang) [reply oceanbasecloud/assets/oauth_clients.json:8]: Hi ,JoshPatel13 . This  domain address  was  temporary unavailable yesterday . It's recovered alreay by now.

**Review by sunzhaoyang — APPROVED** [review]
LGTM

BTW: How I become a reviewer....

File: oceanbasecloud/assets/oauth_clients.json (hunk 1 above)
Comment (JoshPatel13) [reply oceanbasecloud/assets/oauth_clients.json:8]: Sounds good Thanks!

**Review by JoshPatel13 — APPROVED** [review]

**Pull request merged**